supabase/migrations/20231017171314_my_table.sql

```diff
@@ -0,0 +1,54 @@
+drop policy "Enable read access for authenticated users." on "public"."pallet";
+
+drop policy "Enable insert for authenticated users only" on "public"."project";
+
+drop policy "Enable read access for authenticated users." on "public"."shipment";
+
+alter table "public"."pallet" disable row level security;
+
+alter table "public"."pallet_scan" disable row level security;
+
+alter table "public"."project" disable row level security;
+
+alter table "public"."shipment" add column "bookkeeping_flexport" boolean;
+
+alter table "public"."shipment" add column "bookkeeping_gik" boolean;
```

Comment:
Can these both be true / false? Or should we instead have it as a `String` and only allow set values? Or should we just have it as a foreign key of another table called `bookkeeping_systems`:

| id | system   |
|----|----------|
| 1  | flexport |
| 2  | gik      |

Reply:
resolved